doc/nix-local-build.rst

```diff
@@ -11,19 +11,19 @@ local builds with this command (configuring is not necessary):
 
 ::
 
-    $ cabal v2-build
+    $ cabal build
```

Comment:
I think you should strike any changes to this file altogether in the light of the Tom's PR. This will make reviewers' live easier (less files to look at). And also will ease rebasing.

Reply:
Gotcha. I was originally going to fork from his branch but didn't want to juggle multiple the remotes. Didn't even occur to me to just ignore file names he touched. Thanks